src/components/calculator.js

```diff
@@ -1,47 +1,66 @@
-      <CalculatorButton className="digit" character="AC" />
-      <CalculatorButton className="digit" character="+/-" />
-      <CalculatorButton className="digit" character="%" />
-      <CalculatorButton className="digit" character={7} />
-      <CalculatorButton className="digit" character={8} />
-      <CalculatorButton className="digit" character={9} />
-      <CalculatorButton className="digit" character={4} />
-      <CalculatorButton className="digit" character={5} />
-      <CalculatorButton className="digit" character={6} />
-      <CalculatorButton className="digit" character={1} />
-      <CalculatorButton className="digit" character={2} />
-      <CalculatorButton className="digit" character={3} />
-      <CalculatorButton className="digit zero" character={0} />
-      <CalculatorButton className="digit" character="." />
+      <div className="operandSection">
+        <CalculatorButton className="digit" character="AC" handleOnClick={handleClick} />
+        <CalculatorButton className="digit" character="+/-" handleOnClick={handleClick} />
+        <CalculatorButton className="digit" character="%" handleOnClick={handleClick} />
+        <CalculatorButton className="digit" character="7" handleOnClick={handleClick} />
+        <CalculatorButton className="digit" character="8" handleOnClick={handleClick} />
+        <CalculatorButton className="digit" character="9" handleOnClick={handleClick} />
+        <CalculatorButton className="digit" character="4" handleOnClick={handleClick} />
+        <CalculatorButton className="digit" character="5" handleOnClick={handleClick} />
+        <CalculatorButton className="digit" character="6" handleOnClick={handleClick} />
+        <CalculatorButton className="digit" character="1" handleOnClick={handleClick} />
+        <CalculatorButton className="digit" character="2" handleOnClick={handleClick} />
+        <CalculatorButton className="digit" character="3" handleOnClick={handleClick} />
+        <CalculatorButton className="digit zero" character="0" handleOnClick={handleClick} />
+        <CalculatorButton className="digit" character="." handleOnClick={handleClick} />
+      </div>
     </div>
-  </div>
-);
+  );
+};
 
-const CalculatorButton = ({ character, className }) => (
-  <button type="button" className={className}>{character}</button>
+const CalculatorButton = ({ character, className, handleOnClick }) => (
+  <button type="button" className={className} onClick={handleOnClick}>{character}</button>
 );
 
 CalculatorButton.propTypes = {
   character: PropTypes.string.isRequired,
   className: PropTypes.string.isRequired,
+  handleOnClick: PropTypes.func.isRequired,
 };
 
-const OutputDisplay = () => (
-  <div className="outputDisplay">1234567890</div>
+const OutputDisplay = ({ contented }) => (
+  <div className="outputDisplay">{contented}</div>
 );
+
+OutputDisplay.propTypes = {
+  contented: PropTypes.string,
+};
+
+OutputDisplay.defaultProps = {
+  contented: '0',
+};
```

Comment:
- **[OPTIONAL]** I think it would be a good idea to move these components to their own files for better organization